dashboard/app/views/levels/_standalone_video.html.haml

```diff
@@ -13,7 +13,7 @@
 
   - unless @level.video_full_width
     %h1= "#{t('video.label')}: #{video.localized_name}"
-  %div= render(inline: @level.long_instructions, type: :md) if @level.long_instructions
+  %div= render(inline: @level.localized_long_instructions, type: :md) if @level.localized_long_instructions
```

Comment:
This rendering solution unfortunately isn't translator-safe. We need to convert these over to use the clientside renderer